# Undertanding the cartwheel data set



## Importing and inspecting the data

In [1]:
import pandas as pd

path = "datasets/iris/iris.csv"
colnames = ["Sepal_Length", "Sepal_Width","Petal_Length","Petal_Width", "Flower_Name"]
dataset = pd.read_csv(path, names = colnames)

## Basic information (for activity)

In [2]:
dataset

Sepal_Length  Sepal_Width  Petal_Length  Petal_Width     Flower_Name
0             5.1          3.5           1.4          0.2     Iris-setosa
1             4.9          3.0           1.4          0.2     Iris-setosa
2             4.7          3.2           1.3          0.2     Iris-setosa
3             4.6          3.1           1.5          0.2     Iris-setosa
4             5.0          3.6           1.4          0.2     Iris-setosa
..            ...          ...           ...          ...             ...
145           6.7          3.0           5.2          2.3  Iris-virginica
146           6.3          2.5           5.0          1.9  Iris-virginica
147           6.5          3.0           5.2          2.0  Iris-virginica
148           6.2          3.4           5.4          2.3  Iris-virginica
149           5.9          3.0           5.1          1.8  Iris-virginica

[150 rows x 5 columns]

In [3]:
dataset.Flower_Name.value_counts()

Iris-virginica     50
Iris-setosa        50
Iris-versicolor    50
Name: Flower_Name, dtype: int64

# Activity: work with the iris dataset

#### 1. Load the iris.csv file in your computer and compare with the data printed above.

#### 2. How many observations (rows) are in total?
    150

#### 3. How many variables (columns) are in total? What do they represent?
    5

#### 4. How many observatoins are for each type of flower? 
    Iris-virginica     50
    Iris-setosa        50
    Iris-versicolor    50

#### 5. What is the type of data for each variable?
    Everyone quanity type of data is float.

#### 6. What are the units of each variable?
    Not specified